fuzzbench/run_experiment.py

```diff
@@ -24,7 +24,7 @@
 def run_experiment():
     """Main experiment logic."""
     print('Initializing the job queue.')
-    queue = rq.Queue()
+    queue = rq.Queue('build_n_run_queue')
```

Comment:
i was just thinking that 'build_n_run_queue' can be misunderstood as a queue that contains tasks that "build and run" at the same time. maybe we should drop the "n" and just call it "build_run_queue"? this is not really important, but feel free to change it if you think it makes sense

Reply:
Like this point. I think we can make a comment before this line, saying
```
# Create the queue for scheduling build jobs and run jobs.
```
wdyt?

Reply:
sg